themes/src/main/resources/theme/base/login/template.ftl

```diff
@@ -32,7 +32,64 @@
 </head>
 
 <body class="${properties.kcBodyClass!}">
-    <div id="kc-logo"><a href="${properties.kcLogoLink!'#'}"><div id="kc-logo-wrapper"></div></a></div>
+  <div class="${properties.kcLoginClass!}">
+    <header class="${properties.kcHeaderClass!}">
+      <a href="${properties.kcLogoLink!'#'}">
+        <img class="${properties.kcLogoClass!}" src="${url.resourcesPath}/img/keycloak-logo.png" />
```

Comment:
We should never point directly to an image.  Note how the line you deleted has <div id="kc-logo-wrapper">.  This class is in login.css under the Keycloak theme.  See https://github.com/keycloak/keycloak/blob/master/themes/src/main/resources/theme/keycloak/login/resources/css/login.css#L77-L83.  Use kc-logo-wrapper instead of hard-coded keycloak-logo.png.

This will allow customers to use their own logo by overriding login.css.